tooling/v8-snapshot/src/setup/force-no-rewrite.ts

```diff
@@ -69,4 +69,7 @@ export default [
   'node_modules/prettier/parser-meriyah.js',
   'node_modules/prettier/parser-typescript.js',
   'node_modules/prettier/third-party.js',
+  'packages/server/node_modules/is-ci/index.js',
```

Comment:
Why do these need to be in the force-no-rewrite category?

Reply:
I'm trying to figure out exactly why, but it wouldn't work without adding those (they probably don't _all_ need to be in there).

I think what might be happening is that the exported value from `is-ci` is getting evaluated and snapshotted before `process.env.CI=true` is set, and then it never gets re-evaluated? 

If I don't have these in this file then `is-ci` returns `false` even though we are running in CI and it should be `true`. @ZachJW34 originally suggested trying this when we were working through it

Reply:
@astone123 and I paired on this a bit. Not sure we need every one of these but it wasn't working without this addition. I believe it's due to how `is-ci` and `ci-info` are written, where the export is evaluated upon require e.g. `module.exports = !!process.env.CI` and not a function.

Could be totally wrong though!

Reply:
I think from playing around locally all we need to add here is the first two and I'm good with that for now. Though I still want to figure a little more on why.